lib/Parser/RegexCompileTime.cpp

```diff
@@ -4,6 +4,9 @@
 //-------------------------------------------------------------------------------------------------------
 #include "ParserPch.h"
 
+#pragma warning(push)
+#pragma warning(disable:4366)
```

Comment:
How often is this needed? Maybe it's too much, but it might be better to be precise and use __unaligned at the places where it is needed